extensions-core/core/deployment/src/main/java/org/apache/camel/quarkus/core/deployment/CamelRegistryProcessor.java

```diff
@@ -48,6 +49,21 @@ CamelRegistryBuildItem registry(CamelRecorder recorder) {
         return new CamelRegistryBuildItem(recorder.createRegistry());
     }
 
+    /*
+     * TODO: this is a workaround for https://github.com/apache/camel-quarkus/issues/1765
```

Comment:
Could you please file a follow up issue so that we do not forget?

Reply:
done already: https://github.com/apache/camel-quarkus/issues/1769